### ISSUE TITLE
tar: Unexpected EOF in archive

### ISSUE THREAD

**Issue by dmacguigan** [nanoporetech/dorado#969]
Hello,

I am trying to unpack the latest version of Dorado (0.7.3). However, I encounter the following error suggesting that the tarball is corrupted.

```
dmacguig@vortex-future:~/project/software/Dorado$ wget https://cdn.oxfordnanoportal.com/software/analysis/dorado-0.7.3-linux-x64.tar.gz
--2024-08-01 10:56:11--  https://cdn.oxfordnanoportal.com/software/analysis/dorado-0.7.3-linux-x64.tar.gz
Resolving cdn.oxfordnanoportal.com... 18.165.25.79, 18.165.25.52, 18.165.25.24, ...
Connecting to cdn.oxfordnanoportal.com|18.165.25.79|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1307492352 (1.2G) [application/x-tar]
Saving to: ‘dorado-0.7.3-linux-x64.tar.gz’

dorado-0.7.3-linux-x64. 100%[=============================>]   1.22G  32.9MB/s    in 33s     

2024-08-01 10:56:48 (38.2 MB/s) - ‘dorado-0.7.3-linux-x64.tar.gz’ saved [1307492352/1307492352]

dmacguig@vortex-future:~/project/software/Dorado$ tar -zxvf dorado-0.7.3-linux-x64.tar.gz 
dorado-0.7.3-linux-x64/bin/
dorado-0.7.3-linux-x64/bin/dorado
dorado-0.7.3-linux-x64/lib/
dorado-0.7.3-linux-x64/lib/libhdf5.so.8
dorado-0.7.3-linux-x64/lib/libzstd.so
dorado-0.7.3-linux-x64/lib/libcublasLt.so.11
dorado-0.7.3-linux-x64/lib/libnvrtc-builtins.so.11.8
dorado-0.7.3-linux-x64/lib/libnvToolsExt.so.1.0.0
dorado-0.7.3-linux-x64/lib/libnvrtc.so.11.2
dorado-0.7.3-linux-x64/lib/libcublas.so
dorado-0.7.3-linux-x64/lib/libcaffe2_nvrtc.so
dorado-0.7.3-linux-x64/lib/libcublas.so.11.11.3.6
dorado-0.7.3-linux-x64/lib/libnvrtc-builtins.so
dorado-0.7.3-linux-x64/lib/libnvrtc.so.11.8.89
dorado-0.7.3-linux-x64/lib/libcublasLt.so.11.11.3.6

gzip: stdin: unexpected end of file
tar: Unexpected EOF in archive
tar: Unexpected EOF in archive
tar: Error is not recoverable: exiting now
```

**Comment by StephDC** [comment]
Would you mind to do a quick checksum for the downloaded file so we could know if the downloaded file is corrupted or not?

Command: 
```
sha256sum dorado-0.7.3-linux-x64.tar.gz
```

Expected output: 
```
fae832f639aba397399ea36520fb207dd6c2b4c27204586332a99f9fc6114c4f  dorado-0.7.3-linux-x64.tar.gz
```

If you got anything other than the above checksum, please download the file again.

**Comment by sklages** [comment]
@HalfPhoton - probably a good idea to provide the checksums along with the download files ..

**Comment by HalfPhoton** [comment]
I'll add it to the list of enhancements, thanks @sklages

**Comment by dmacguigan** [comment]
I know this issue is already closed, but the new tarball for 0.7.3 works fine now
